teams/teams-ps/teams/New-CsCustomPolicyPackage.md

```diff
@@ -0,0 +1,104 @@
+---
+external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
+Module Name: MicrosoftTeams
+title: New-CsCustomPolicyPackage
+author: gucsun
+ms.author: gucsun
+manager: amitar
+online version: https://docs.microsoft.com/powershell/module/teams/new-CsCustomPolicyPackage
+schema: 2.0.0
+---
+
+# New-CsCustomPolicyPackage
+
+## SYNOPSIS
+This cmdlet submits an operation that creates a custom policy package with custom package name, description and a list of policies.
+
+## SYNTAX
+
+```
+New-CsBatchPolicyPackageAssignmentOperation -Identity <String> -PolicyList <String[]> [-Description <String>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet submits an operation that creates a custom policy package. It allows the user to create their own policy package. For more information on policy packages and the policy types available, please review https://docs.microsoft.com/MicrosoftTeams/manage-policy-packages.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> New-CsCustomPolicyPackage -Identity myCustomPackage -PolicyList "TeamsMeetingPolicy, Education_Teacher" , "TeamsMessagingPolicy, Firstline_Manager" -Description "My first custom package"
+```
+
+Creates a custom package named "myCustomPackage" with two policies included: TeamsMeeting policy named "Education_Teacher" and TeamsMessaging policy named "Firstline_Manager". Describe it as "My first custom package".
```

Comment:
can be rephrased:
Creates a custom package named "myCustomPackage" with description "My first custom package"...